README.md

```diff
@@ -402,7 +402,7 @@ _Type: Boolean_
 _Default: n_
 
 If set to `y` and SSL support is not compiled into Python, the agent will fall-back to
-use HTTP. Otherwise, if SSL support is not compiled into Python, the agent will fail
+use HTTP. Otherwise, if SSL support is compiled into Python, the agent will fail
```

Comment:
I think the current documentation is correct, though a little hard to read. Maybe we can re-phrase it to 

> If SSL support is not compiled into Python, the agent will fail all HTTPS requests. You can set this option to 'y' to make the agent fall-back to HTTP, instead of failing the requests.

?